tools/validate_configs/test_configs/text_escape.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: text_escape
+
+vars:
+  project_id:  #
+  deployment_name: text_escape
+  region: us-east4
+  zone: us-east4-c
+  labels:
+    red: \((blue))
+
+deployment_groups:
+- group: zero
```

Comment:
I don't believe having multiple groups is related to text escaping and we already have a golden test for igc. I would prefer to keep the scope smaller to only test what we claim to be testing: text escaping.

Please simplify the blueprint to have a single group and focus test around text escaping.